new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/TestAssertions.java

```diff
@@ -187,4 +187,18 @@ public static boolean doesImageExist(String searchString) {
   public static boolean isHelmReleaseDeployed(String releaseName, String namespace) {
     return Helm.isReleaseDeployed(releaseName, namespace);
   }
+
+  /**
+   * Check if a persistent volume is in a given state.
+   *
+   * @param name name of the persistent volume
+   * @param state state of the persistent volume to match
+   * @return true if state matches otherwise false
+   */
+  public static Callable<Boolean> isPersistentVolumeInState(String name, String state) {
```

Comment:
feels like an enum would make sense for the state arg

Reply:
see my above comment